ga4gh/cli/repomanager.py

```diff
@@ -1,6 +1,8 @@
 """
 repo manager cli
 """
+
+
```

Comment:
Oh I did that for some particular/intentional reasons.  I would put a single space between lines of code in a function (sometimes I won't even do that! - a 1-line hash comment can most times be read _as_ a space between source code lines), and I would use 2 spaces to separate top-level functions, as well as all of the other top-level things in a single Python file, like the top comment, constant/global/capitalized variables, and any initialization code that's not in a top-level function, including the `if __name__ == '__main__'` stuff at the bottom.

That's what most Python files I've ever seen looked like, and in that order: the top comment, the import lines, global variables and initialization code (like `logger = logging.getLogger()`), a mix of top-level function and class definitions, and the usual `if __name__ == '__main__'` stuff at the bottom if you want.  And then I've grown over time to separate those _large_ things by 2 spaces, and within those large things, separate the __small__ things you're working with there with 1 space.

But sure, I'll go ahead and make the changes.